specification/vnfmanager/resource-manager/Microsoft.VnfManager/preview/2020-01-01-preview/vendor.json

```diff
@@ -0,0 +1,867 @@
+          },
+          {
+            "$ref": "./vnfmanager.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./vnfmanager.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Update successful. The operation returns the resulting Vendor Sku subresource.",
+            "schema": {
+              "$ref": "#/definitions/VendorSku"
+            }
+          },
+          "202": {
+            "description": "Create successful. The operation returns the resulting Vendor Sku subresource.",
+            "schema": {
+              "$ref": "#/definitions/VendorSku"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./vnfmanager.json#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or update the sku of Vendor resource": {
+            "$ref": "./examples/VendorSkuCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VnfManager/vendors/{vendorName}/skus": {
+      "get": {
+        "tags": [
+          "Vendors"
+        ],
+        "operationId": "VendorSkus_List",
+        "description": "Gets all the skus in a Vendor.",
+        "parameters": [
+          {
+            "name": "vendorName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the vendor."
+          },
+          {
+            "$ref": "./vnfmanager.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./vnfmanager.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of Vnf Manager Vendor resources.",
+            "schema": {
+              "$ref": "#/definitions/VendorSkuListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./vnfmanager.json#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get all the skus of Vendor resource": {
+            "$ref": "./examples/VendorSkuListAll.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VnfManager/locations/{locationName}/vendors/{vendorName}/vnfs/{vnfServiceKey}": {
```

Comment:
>/subscriptions/{s [](start = 5, length = 17)

for all tracked resources, define 'patch' that allows updating at least the tags

Reply:
Vendor and sku are the proxy resource. these does not need any tags update.